docs/guides/databases/mariadb/backup-mariadb-mysql-to-object-storage-with-restic/index.md

```diff
@@ -53,45 +55,59 @@ The steps in this guide require root privileges. Be sure to run the steps below
 
 ## Install Restic
 
-{{< content "applications/backups/install-restic-shortguide" >}}
+{{< content "install-restic-shortguide" >}}
 
 ## Create the Restic Repository
 
-{{< content "applications/backups/create-restic-repository-shortguide" >}}
+{{< content "create-restic-repository-shortguide" >}}
 
-## Set Up Automated Database Backups
+## Backup All Databases
 
 {{< note >}}
-In each of the below commands, remember to replace "your-bucket-name" and "us-east-1.linodeobjects.com" with the name of your Object Storage bucket and cluster hostname.
+In each of the below commands, remember to replace `your-bucket-name` and `us-east-1.linodeobjects.com` with the name of your Object Storage bucket and cluster hostname.
 {{< /note >}}
 
-### Backup all databases
-
 The mysqldump utility is used to dump the contents of a database to a SQL file on-disk on your Linode. This simple shell script will loop through all
 databases on your server, and dump each one to its own SQL file.
 
-Create a file in /usr/local/bin:
+1. Create a file in `/usr/local/bin`:
 
-    sudo nano /usr/local/bin/backup_mariadb
+        sudo nano /usr/local/bin/backup_mariadb
 
-Copy the following contents into the file:
+1. Copy the following contents into the file:
 
-{{< file "/usr/local/bin/backup_mariadb" >}}#!/bin/bash
+    {{< file "/usr/local/bin/backup_mariadb" >}}
+#!/bin/bash
+PATH="/usr/local/bin:$PATH"
 source /root/restic_params
-mysql -N -e 'show databases' | while read dbname; do /usr/bin/mysqldump --complete-insert "$dbname" > "/var/backups/mariadb/$dbname".sql; done
+mysql --defaults-extra-file=/root/mysql_cnf -N -e 'show databases' | while read dbname; do /usr/bin/mysqldump --defaults-extra-file=/root/mysql_cnf --complete-insert "$dbname" > "/var/backups/mariadb/$dbname".sql; done
 restic -r s3:us-east-1.linodeobjects.com/your-bucket-name -p /root/restic_pw backup /var/backups/mariadb
 {{< /file >}}
 
-Make the script executable and create the folder to hold the backup files (if it doesn't already exist):
+1. Make the script executable and create the folder to hold the backup files (if it doesn't already exist):
 
-    chmod u+x /usr/local/bin/backup_mariadb
-    mkdir -p /var/backups/mariadb/
+        sudo chmod u+x /usr/local/bin/backup_mariadb
```

Comment:
The limited user I was running needed to use `sudo` here (and on the other lines where I added it)